docs/docs/troubleshooting.md

```diff
@@ -0,0 +1,98 @@
+---
+id: troubleshooting
+title: Troubleshooting
+sidebar_title: Troubleshooting
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const OsTabs = (props) => (<Tabs
+groupId="operating-systems"
+defaultValue="debian"
+values={[
+{label: 'Debian/Ubuntu', value: 'debian'},
+{label: 'Raspberry OS', value: 'raspberryos'},
+{label: 'Fedora', value: 'fedora'},
+{label: 'Windows', value: 'win'},
+{label: 'macOS', value: 'mac'},
+]
+}>{props.children}</Tabs>);
+
+### Environment Variables not Defined
```

Comment:
Before diving into a specific troubleshooting item, can we please add a `## Summary` section that outlines the purpose of the doc, and links to the Discord as a resource if the troubleshooting page doesn't help resolve the issue?